### PR TITLE
Scripts: Code refractor For function `parseRequestedNames`

### PR DESCRIPTION
**Pull request by Biki-das**
This change is just a little code refractor I tried , this surely makes the code more easy to read , and we have few variables than earlier

### CODE REVIEW

**Comment by Biki-das** [comment]
@gaearon what do you think about this?

**Comment by Biki-das** [comment]
@lunaruan could you review this?

**Comment by Biki-das** [comment]
@gaearon , does this help any way?

File: scripts/rollup/build.js [hunk 1]
```diff
@@ -66,23 +66,20 @@ const {
...
+    .join()
+    .split(',')
+    .map(name => {
+      name = name.trim();
       if (toCase === 'uppercase') {
-        name = name.toUpperCase();
+        return name.toUpperCase();
       } else if (toCase === 'lowercase') {
-        name = name.toLowerCase();
+        return name.toLowerCase();
```
Comment (shanigettacar) [inline scripts/rollup/build.js:77]: You don't need else after return

File: scripts/rollup/build.js [hunk 2]
```diff
@@ -66,23 +66,20 @@ const {
...
+      } else {
+        return name;
       }
-      result.push(name);
-    }
-  }
-  return result;
+    })
+    .filter(name => !!name);
```
Comment (shanigettacar) [inline scripts/rollup/build.js:82]: I think this filter should happen before the map

**Pull request closed without merging**